Classes/ViewHelpers/Security/AssetAccessViewHelper.php

```diff
@@ -59,9 +60,11 @@ public function render(Folder $folder, File $file = null)
      * @param array $arguments
      * @return bool
      */
-    protected static function evaluateCondition($arguments = null)
+    protected static function evaluateCondition(array $arguments)
```

Comment:
this signature is different in TYPO3 7 and also in \TYPO3Fluid\Fluid\Core\ViewHelper\AbstractConditionViewHelper